src/utils.ts

```diff
@@ -29,44 +29,84 @@ function hasScope(file: string) {
   return file.match(/^@/);
 }
 
-export async function findPackages(storagePath: string, validationHandler: Function) {
-  const listPackages: Array<any> = [];
+async function getVersions(scopePath: string, packageName: string): Promise<string[]> {
+  const versions: string[] = [];
+  const arr: string[] = await readDirectory(scopePath);
+  arr.forEach( (filePath: string) => {
+    const fileName = path.basename( filePath);
+    // fileName should start with package name and should ends with '.tgz'
+    if (fileName.indexOf(packageName)===0 && fileName.indexOf('.tgz')!==-1) {
+      const v: string = (fileName.split(packageName+'-')[1] || '').replace('.tgz', '');
+      versions.push(v);
+    }
+  });
+  return versions;
+}
+
+
+export async function findPackages(storagePath: string) {
+  //stats
+  const startTS = Date.now();
+  let packages_count = 0;
```

Comment:
we use `camelCase` format for consistency.

Reply:
Fixed